src/Nethermind/Nethermind.Evm/Tracing/ITxTracer.cs

```diff
@@ -127,6 +127,8 @@ public interface ITxTracer : IWorldStateTracer
         /// </remarks>
         bool IsTracingFees { get; }
 
+        bool IsTracing => IsTracingReceipt || IsTracingActions || IsTracingOpLevelStorage || IsTracingMemory || IsTracingInstructions || IsTracingRefunds || IsTracingCode || IsTracingStack || IsTracingBlockHash || IsTracingAccess || IsTracingFees;
+
```

Comment:
Potentially this could be calculated once?

Reply:
Not sure if we had some confusion here?, this was fine in interface if not calculated once per tracer (instead of calculating it in every call)

Reply:
Was in default implementation before so would be calculated every call and make 13 interfaces calls per call as the default implementation. Moving it into the tracers makes it 1 interface call

Reply:
Ok can it be calculated once instead per call in some of them?

Reply:
Where they are consts the Jit will fold them into a const anyway since simple inlines now (not going through the interface)